runtime/src/main/jni/include/v8-debug.h

```diff
@@ -8,19 +8,19 @@
 #include "v8.h"  // NOLINT(build/include)
 
 /**
- * Debugger support for the V8 JavaScript engine.
+ * ATTENTION: The debugger API exposed by this file is deprecated and will be
+ *            removed by the end of 2017. Please use the V8 inspector declared
+ *            in include/v8-inspector.h instead.
```

Comment:
Cool, we need to remove the dependencies from the debugger: just making it here.